### ISSUE TITLE
Documentation on log messages

### ISSUE THREAD

**Issue by adiell** [catboost/catboost#1130]
Problem:
The documentation is lacking in respect for the log messages.
For example log in info level yields messages of the following kind:
```
{feature names} pr0 tb0 type0, border=6 score 65.48010581
```
What are `pr0`, `tb0` etc?
How to read the log in general.
Thanks!

**Comment by annaveronika** [comment]
Those are ways to understand which split has been selected for a categorical feature.
- pr0 means prior with index 0. By default classification uses 3 constant priors, 0, 0.5 and 0. This feature has prior with index 0, which is equal to 0.
- tb0 means that target border with index 0 is selected. For binary classification there is only one target border 0.5, which splits target into values less then 0.5 (0) and greater than 0.5 (1).
- border=6 is the value of the border for the numerical feature that is calculated using the categorical one. The samples with feature < 6 go to the left in the tree, and the samples with feature >= 6 go to the right of the split.
- score is the value of the score that this split was assigned. The split with biggest score is always selected.

We will add documentation for that at some point, but for now please use the default logging, which doesn't have all this info. You probably don't need it anyway.

**Comment by LucaChech** [comment]
Is this still open? If so, I can go ahead with it.

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@LucaChech Yes, log messages are still ugly for categorical splits. We would be very grateful if you fixed this in a pull request 😻
